cg2019_paper.tex

```diff
@@ -580,7 +581,7 @@ \section{Experiments and results}
 
 New pole position selection has been conducted considering \textit{Alpine 2} track, 5 races and 20 laps per race. 
 We have defined a score function based in Formula 1 schema, so the obtained punctuations depend on the car position in the final rank: 1 - 25 points, 2 - 18, 3 - 15, 4 - 12, 5 - 10, 6 - 8, 7 - 6, 8 - 4, 9 - 2, 10 - 1. The the starting grid (initial positions of cars) on these races was set randomly.
-Due to the high-demanding time this method is, this race-based selection process has just been performed every 5 generations (not in all of them).
+Due to the \textcolor{red}{high time-demanding nature of this method}, this race-based selection process has just been performed every 5 generations (not in all of them).
```

Comment:
probably drop high?

Reply:
Probably best. I didn't dare because I wasn't sure of the importance of stressing "high".

Reply:
Please do the honors.